modules/overriding-default-node-ip-selection-logic.adoc

```diff
@@ -0,0 +1,142 @@
+// This is included in the following assemblies:
+//
+// * installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+
+:_content-type: PROCEDURE
+[id="overriding-default-node-ip-selection-logic_{context}"]
+= Optional: Overriding the default node IP selection logic 
+
+In most circumstances, the `NODEIP_HINT` is the API VIP. The IP selection logic will determine which subnet the `NODEIP_HINT` belongs to, and from that determine the interface in that subnet and select an IP on that interface. For example, a node that has two interfaces, `eth0` with the subnet range `10.0.0.10/24`, and `eth1` with the subnet range `192.0.2.5/24`, and with a default route via the `10.0.0.10` source address on `eth0`:
+
+.Example
+
+[source,terminal]
+----
+$ ip route show
+----
++
+Example output:
++
+[source,terminal]
+----
+default via 10.0.0.1 dev eth0 proto dhcp src 10.0.0.10 metric 426
+10.0.0.0/24 dev eth0 proto kernel scope link src 10.0.0.10 metric 48
+192.0.2.0/24 dev eth1 proto kernel scope link src 192.0.2.5 metric 48
+----
+
+By default, the `10.0.0.10` IP address is selected for use by the node.
+
+To override the default IP selection logic, you can create a hint file that includes the `NODEIP_HINT` variable to override the default IP selection logic. Creating a hint file allows users to select a specific node IP address from the interface in the subnet of the IP address specified in the `NODEIP_HINT` variable. For example, users can configure the `NODEIP_HINT` variable to point at an IP of `192.0.2.1` so that the correct subnet, `192.0.2.0/24`, is selected. As a result, the `192.0.2.5` IP address on `eth1` is used for the node. 
+
+Use the following procedure to override the default node IP selection logic. 
+
+.Procedure
+
+. Reveal the available IP routes of the node by entering the following command: 
++
+[source,terminal]
+----
+$ ip route show
+----
++
+Example output:
++
+[source,terminal]
+----
+default via 10.0.0.1 dev eth0 proto dhcp src 10.0.0.10 metric 426
+10.0.0.0/24 dev eth0 proto kernel scope link src 10.0.0.10 metric 48
+192.0.2.0/24 dev eth1 proto kernel scope link src 192.0.2.5 metric 48
+----
+
+. Use the `runtimecfg` command to reveal the interface that can reach the IP routes:
++
+[source,terminal]
+----
+$ runtimecfg 192.0.2.1 
+----
++
+Example output:
++
+[source,terminal]
+----
+192.0.2.1 dev eth1 src 192.0.2.5 uid 1000
+    cache
+----
+
+. Add a hint file to your your `/etc/default/nodeip-configuration` file. The contents of your hint file must match the following example: 
++
+[source,text]
+----
+NODEIP_HINT=192.0.2.5
+----
++
+[IMPORTANT]
+====
+* Do not use the exact IP address of a node as a hint, for example, `192.0.2.1`. Using the exact IP address of a node causes the node using the hint IP address to fail to configure correctly.
```

Comment:
I think this example is wrong too. It should be 192.0.2.5 (assuming this is supposed to be an example of an incorrect value).